compiler/circle-tensordump/driver/Driver.cpp

```diff
@@ -25,66 +26,41 @@
 #include <string>
 #include <vector>
 
-using OptionHook = std::function<std::unique_ptr<circletensordump::DumpInterface>(void)>;
-
-void print_help(const char *progname)
-{
-  std::cerr << "USAGE: " << progname << " options circle [hdf5]" << std::endl;
-  std::cerr << "   --tensors : dump tensors in circle file" << std::endl;
-  std::cerr << "   --tensors_to_hdf5 : dump tensors in circle file to hdf5 file" << std::endl;
-}
-
 int entry(int argc, char **argv)
 {
-  if (argc < 3)
-  {
-    std::cerr << "ERROR: Failed to parse arguments" << std::endl;
-    std::cerr << std::endl;
-    print_help(argv[0]);
-    return EXIT_FAILURE;
-  }
+  arser::Arser arser{
+      "circle-tensordump allows users to retrieve tensor information from a Circle model file"};
 
-  // Simple argument parser (based on map)
-  std::map<std::string, OptionHook> argparse;
+  arser.add_argument("circle").nargs(1).type(arser::DataType::STR).help("Circle file path to dump");
```

Comment:
Just curious, not using `--circle` is intended?

Reply:
@llFreetimell Yes. Double dash is for optional argument. And without it is for positional argument.